glean_parser/validate_ping.py

```diff
@@ -23,18 +23,9 @@
 SCHEMAS_DIR = ROOT_DIR / "schemas"
 
 
-PING_SCHEMA_DEFAULT_URL = (
-    "https://raw.githubusercontent.com/mozilla-services/"
-    "mozilla-pipeline-schemas/dev/schemas/glean/baseline/"
-    "baseline.1.schema.json"
-)
-
-
 @functools.lru_cache(maxsize=1)
 def _get_ping_schema(schema_url):
-    contents = util.fetch_remote_url(
-        schema_url, cache=(schema_url != PING_SCHEMA_DEFAULT_URL)
-    )
+    contents = util.fetch_remote_url(schema_url)
```

Comment:
We could just have `cache=True` here (and reinstate the caching behavior above).